content/workshops/openshift_service_mesh/lab3.3_jaeger.md

```diff
@@ -138,7 +139,7 @@ for ((i=1;i<=5;i++)); do curl -s -o /dev/null $GATEWAY_URL/profile; done
 Inspect the traces.  
 <blockquote>
 <i class="fa fa-desktop"></i>
-On the left bar under 'Search', select 'app-ui.microservies-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local'for 'Operation'.  
+On the left bar under 'Search', select 'app-ui.microservices-demo' for 'Service' and 'userprofile-microservices-demo.svc.cluster.local' for 'Operation'.  
```

Comment:
Let's switch this to '{service-name}.userX' to be in sync with what workshop users are looking at.